src/hooks/useDailyStatistics/useDailyStatistics.test.tsx

```diff
@@ -7,6 +7,7 @@ import { defaultIdentifier } from "../../test/helpers/defaults";
 import { CampaignPolicy } from "../../types";
 import { Sentry } from "../../utils/errorTracking";
 import { wait } from "@testing-library/react-native";
+import "../../common/i18n/i18nMock";
```

Comment:
hmm, are we using this import here? if we're not using it, we can remove it haha (:

Reply:
we need that because of mocking. if not the test will fail.

Reply:
I don't see anything being used here. What is being used from this import?

Reply:
the purpose of importing that is to set the default translation locale, if not error will be thrown when translating texts.

Reply:
i see, okok